adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -642,6 +642,29 @@ internal IWebUI CreateWebAuthenticationDialog(PlatformParameters parameters)
                     new ClientKey(clientAssertion), resource).ConfigureAwait(false);
         }
 
+#if !(ANDROID || iOS || WINDOWS_APP)
+        /// <summary>
+        /// Acquires security token from the authority using an authorization code previously received.
+        /// This method does not lookup token cache, but stores the result in it, so it can be looked up using other methods such as <see cref="AuthenticationContext.AcquireTokenSilentAsync(string, string, UserIdentifier)"/>.
+        /// </summary>
+        /// <param name="authorizationCode">The authorization code received from service authorization endpoint.</param>
+        /// <param name="redirectUri">The redirect address used for obtaining authorization code.</param>
+        /// <param name="clientCertificate">The client certificate to use for token acquisition.</param>
+        /// <param name="resource">Identifier of the target resource that is the recipient of the requested token. It can be null if provided earlier to acquire authorizationCode.</param>
+        /// <param name="sendX5c">This parameter enables application developers to achieve easy certificates roll-over
```

Comment:
sendCertificate instead of sendX5c (everywhere). This is a breaking change in ADAL.NET for existing methods, but since we are going to release a preview of a new major version, I guess this is ok
(breaking change as customers could have used the sendX5c : true syntax)

Reply:
Resolved